content/en/blog/porterlb-boosts-kubernetes-on-bare-metal.md

```diff
@@ -0,0 +1,61 @@
+
+## Layer 2 Mode
+
+Generally, you are advised to use the BGP mode to expose your Services in a high availability network. However, BGP may be unavailable in certain systems because of security requirements or because the router does not support BGP. In this case, you can use PorterLB in Layer 2 mode to achieve similar functionality.
+
+In Layer 2 mode, PorterLB uses ARP packets (for IPv4) or NDP packets (for IPv6) to map the Service IP address to the MAC address of a Kubernetes node. The mechanism of the Layer 2 mode is similar to that of the BGP mode, except that BGP is replaced with ARP/NDP and the router obtains only one route destined for the Service. 
+
+![porter-layer-2-topology](/images/blog/porterlb-boosts-kubernetes-on-bare-metal/porter-layer-2-topology.jpg)
+
+Though the Layer 2 mode does not provide the same high availability as the BGP mode does, it is easier to use (you don't even need to configure BGP properties):
+
+1. Install PorterLB.
+2. Configure an IP address pool by using Eip.
+3. Create a Service and set the Service to use PorterLB.
+
+You can obtain detailed guidance on how to install, configure, and use PorterLB from the [PorterLB documentation](https://porterlb.io/docs/).
+
+## Advantages of PorterLB
+
+There are other load balancer implementations such as MetalLB designed for bare-metal Kubernetes clusters. Compared with others, PorterLB has the following advantages.
+
+### Cloud native
+
+To manage IP address pools and BGP properties for PorterLB, you only need to use the `kubectl apply` command provided by Kubernetes to create CRD objects. To obtain the status information about IP address pools and BGP peers, you can simply run `kubectl get` to view the status of the CRD objects. No other configuration files are required. In addition, a GUI of PorterLB is in progress and will be available soon, which further simplifies the usage of PorterLB.
+
+### Simple architecture
+
+After PorterLB is installed in a Kubernetes cluster, a porter-manager Deployment that contains a porter-manager Pod is created. The porter-manager Pod implements the functionality of PorterLB for the entire Kubernetes cluster. For high availability, you can scale the porter-manager Deployment and assign multiple PorterLB replicas (porter-manager pods) to multiple cluster nodes. This simple architecture ensures that PorterLB can be easily managed and integrated with other systems.
+
+### Seamless integration with KubeSphere
+
+KubeSphere is a successful open-source container platform and provides enterprise-grade features to simplify development and operations, such as observability and troubleshooting, unified monitoring and logging, centralized storage and networking management, and easy-to-use CI/CD pipelines. KubeSphere adopts a plug-and-play architecture to integrate third-party applications. As a sub-project of KubeSphere, PorterLB fits well into the KubeSphere ecosystem and can be seamless integrated as a plugin with KubeSphere.
```

Comment:
`successful` is strange if you describe an open source product.

Reply:
Title changed and other issues fixed.